cas-server-support-osf/src/main/java/io/cos/cas/authentication/handler/support/OpenScienceFrameworkPrincipalFromRequestRemoteUserNonInteractiveCredentialsAction.java

```diff
@@ -137,6 +141,10 @@ public String getInstitutionId() {
 
     private static final int SIXTY_SECONDS = 60 * 1000;
 
+    private static final String PROTOCOL_SAML = "SHIBBOLETH";
```

Comment:
- [ ] Document the delegation protocols:
  - The protocol specification: CAS, SAML2, OAuth2
  - The protocol implementation: cas-pac4j, saml2-shibboleth, cas-oauth2
  - Version different and sub-protocols: CAS protocol uses CAS10, CAS20, CAS30, SAML11 for `Service Validation`